reactDashboard/src/components/DataCharts/DataCharts.jsx

```diff
@@ -225,6 +226,7 @@ const DataCharts = ({
 
     const REGION_DATASET_COLOUR = "#ec6730";
     const AVERAGE_DATASET_COLOUR = "#767676";
+    const AVERAGE_DATASET_FILL_COLOUR = darkmode ? "rgb(118, 118, 118, 0.50)" : "rgb(118, 118, 118, 0.25)";
```

Comment:
Is this used ?

Reply:
yes

Reply:
Ah, it is now ;)

Reply:
So the latest version has removed the fill area for the population average line on the regional charts. Is this intentional ?

Reply:
No it's back now 😎

Reply:
I have added back in